frontend/src/pages/forms/Form.jsx

```diff
@@ -130,7 +130,7 @@ const Form = ({ walletId, setWalletId }) => {
         </div>
         <div>
           <div className="form-button-container">
```

Comment:
can you please remove this container, seems like this is broken and adds some hover

Reply:
Oh, i found the issue, the button container is valid, i have removed the hover states, so it should work just fine now after this push